src/shared/src/components/molecules/AltinnMobileTableItem.tsx

```diff
@@ -164,6 +178,19 @@ const useStyles = makeStyles({
   aboveEditingRow: {
     borderBottom: 0,
   },
+  popoverButtonContainer: {
+    display: 'flex',
+    '& button': {
+      margin: '1rem 1rem 0 0',
+      '&focus': {
+        outline: 'unset',
+      },
+    },
+  },
```

Comment:
Same comment here as in `RepeatingGroupTable.tsx`. Perhaps this should even be extracted to a separate component, to avoid having to duplicate the styles and logic here?